extra/1-radio-stations.js

```diff
@@ -15,6 +15,14 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies() {
+  let frequencies = [];
+  for (let i = 87; i <= 108; i++) {
```

Comment:
I just suggest  that you can use i < 109 instead of i <= 108,

Reply:
I personally like how Joanna has done it. The task states it should be between 87 - 108 so the code is clearer that it uses 108 not 109, in my opinion.